src/lib/transaction_snark/test/verification_key_permission/verification_key_permission.ml

```diff
@@ -45,7 +41,89 @@ struct
   let is_non_zkapp_update = true
 end
 
-let%test_module "Set verification key permission to Impossible" =
+let%test_module "Update account verification key permission from mainnet to \
+                 berkeley" =
   ( module struct
+    let proof_cache =
+      Result.ok_or_failwith @@ Pickles.Proof_cache.of_yojson
+      @@ Yojson.Safe.from_file "proof_cache.json"
+
+    let () = Transaction_snark.For_tests.set_proof_cache proof_cache
+
+    open Currency
+    open Signature_lib
+    module U = Transaction_snark_tests.Util
+    module Spec = Transaction_snark.For_tests.Update_states_spec
+    include Transaction_snark_tests.Test_zkapp_update.Make (Test_input1)
```

Comment:
Won't we overwrite this line's `include` by the next `include`?

(Asking before looking what's inside that inclusion)

Reply:
Ah. That's an issue. My intention was to have both of them in one test_module so that the cache applies to both.

Reply:
I ended up remove one of them. Since we don't have special rules for change the permissions, there's no need to do two different test on those.